plugins/auth-backend/api-report.md

```diff
@@ -706,7 +706,7 @@ export type TokenParams = {
   claims: {
     sub: string;
     ent?: string[];
-  };
```

Comment:
I'm thinking this should probably be `Record<string, JsonValue>`?

Reply:
Great point.

Reply:
Done